### DISCUSSION TITLE
Keynote Speaker Mario Rodriguez - VP of Product GitHub

### DISCUSSION THREAD

**Discussion by AndreaGriffiths11** [gittogethers/octogatosconf22#131] [Talks in English]
**Mario Rodriguez** is a VP of Product Management at GitHub, currently focusing on all things GitHub Issues. Prior to this focus he led the GitHub Enterprise team, and worked at Microsoft leading Azure DevOps Platform and Artifacts. Mario’s core identity is being a learner and time away from product engineering is spent with his wife and two daughters.

**Comment by AndreaGriffiths11** [comment]
Amazing Keynote Mario!

**Comment by 0xSaikat** [comment]
ok

**Comment by sarkardeep** [comment]
Yes , Amazing Keynote Mario!

**Comment by AndreaGriffiths11** [comment]
In case you missed the entire keynote will be posted to YouTube

**Reply by sarkardeep** [reply]
Official Github You tube :- https://www.youtube.com/c/GitHub

**Reply by 0xSaikat** [reply]
https://www.youtube.com/watch?v=UVdod6kiPcs

**Comment by AliOsman1995** [comment]
Wonderful
